dashboard/app/templates/mail_ns_coverage.txt

```diff
@@ -0,0 +1,4 @@
+Regressions happened in '{{.Namespace}}' from {{.PeriodFrom}} ({{.PeriodFromDays}} days) to {{.PeriodTo}} ({{.PeriodToDays}} days).
```

Comment:
Please add a test that hardcodes a report example, otherwise it's hard to see how it will look like in real life (what are all of these days, line lengths, etc).

It's also easy to make template to panic if some argument name is mistyped. Without tests it will cause pretty bad failure mode now (since called from a cron handler once per month).